config/all-in-one/frr-k8s.yaml

```diff
@@ -307,6 +307,54 @@ spec:
                 required:
                 - routers
                 type: object
+              nodeSelector:
+                description: A label selector is a label query over a set of resources.
+                  The result of matchLabels and matchExpressions are ANDed. An empty
+                  label selector matches all objects. A null label selector matches
+                  no objects.
```

Comment:
maybe we make the nodeSelector description more specific.
i.e. "A nodeSelector is a... over a set of nodes... match all nodes... match no nodes"

Reply:
this is the default description for a `metav1.LabelSelector` field, added a description that matches our usecase